server/superdesk/io/aap.py

```diff
@@ -24,6 +24,9 @@
 
 logger = logging.getLogger(__name__)
 PROVIDER = 'aap'
+errors = [ParserError.nitfParserError(None, None).get_error_description(),
+          ProviderError.ingestError(None, None).get_error_description(),
+          ParserError.parseFileError(None, None, None, None).get_error_description()]
```

Comment:
This looks ugly, maybe we can rid of them or use named parameters so we know to what we are passing `None`